--- gitlab-workhorse-0.7.1/Makefile~	2016-03-21 14:04:24.000000000 +0200
+++ gitlab-workhorse-0.7.1/Makefile	2016-04-17 22:52:07.310556758 +0300
@@ -1,6 +1,6 @@
 PREFIX=/usr/local
 VERSION=$(shell git describe)-$(shell date -u +%Y%m%d.%H%M%S)
-GOBUILD=go build -ldflags "-X main.Version=${VERSION}"
+GOBUILD=go build -ldflags "-X main.Version ${VERSION}"
 
 all: gitlab-zip-cat gitlab-zip-metadata gitlab-workhorse
 
